CHANGELOG.md

```
@@ -1,45 +1,31 @@
+## Bug Fixes
 
 * **#144:** env and extract hardcoded dsn to env ([#151](https://github.com/VilnaCRM-Org/user-service/issues/151)) ([5af27cf](https://github.com/VilnaCRM-Org/user-service/commit/5af27cfec314106bafa2b5c4652267289f8f1486)), closes [#144](https://github.com/VilnaCRM-Org/user-service/issues/144) [#144](https://github.com/VilnaCRM-Org/user-service/issues/144) [#144](https://github.com/VilnaCRM-Org/user-service/issues/144) [#144](https://github.com/VilnaCRM-Org/user-service/issues/144) [#144](https://github.com/VilnaCRM-Org/user-service/issues/144) [#144](https://github.com/VilnaCRM-Org/user-service/issues/144)
 
+## [2.4.0](https://github.com/VilnaCRM-Org/user-service/compare/v2.3.1...v2.4.0) (2025-02-11)
 
-
-# [2.4.0](https://github.com/VilnaCRM-Org/user-service/compare/v2.3.1...v2.4.0) (2025-02-11)
-
-
-### Features
+## Features
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)